app/routers/scraping.py

```
@@ -0,0 +1,348 @@
+        logger.error(f"Unexpected scraping error: {e}")
+        raise HTTPException(
+            status_code=500,
+            detail=f"Internal scraping error: {str(e)}"
+        )
+
+
+@router.post("/jobs/background", response_model=dict)
+async def scrape_jobs_background(
+    request: ScrapeJobsRequest,
+    background_tasks: BackgroundTasks,
+    db: Session = Depends(get_db)
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)